src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -173,8 +173,12 @@ internal partial class CodeFixService : ICodeFixService
             // invariant: later code gathers & runs CodeFixProviders for diagnostics with one identical diagnostics span (that gets set later as CodeFixCollection's TextSpan)
             // order diagnostics by span.
             SortedDictionary<TextSpan, List<DiagnosticData>>? aggregatedDiagnostics = null;
+
+            // Only execute analyzers with fixable diagnostics for 'CodeActionPriorityRequest.Normal'
```

Comment:
why?

Reply:
I'll clarify in the comment, but we do so now as Normal priority bucket would never have a code action for diagnostics without a corresponding code fixer. These analyzers can only have suppression/configuration code actions, which are part of Lowest priority bucket.